modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -267,6 +270,9 @@ export default {
         menuPlacement: 'top'
       };
     },
+    widgetIcon() {
+      return this.contextMenuOptions.menu.filter(item => item.name === this.widget.type)[0].icon || 'shape-icon';
```

Comment:
```suggestion
      return this.contextMenuOptions.menu.filter(item => item.name === this.widget.type)[0]?.icon || 'shape-icon';
```

if we ever be in the case where `filter` returns no item:

<img width="888" alt="image" src="https://github.com/apostrophecms/apostrophe/assets/8301962/6666e0e5-7ab4-4da4-8c19-193e0816460a">